base/sharedarray.jl

```diff
@@ -24,6 +24,17 @@ type SharedArray{T,N} <: DenseArray{T,N}
     SharedArray(d,p,r,sn) = new(d,p,r,sn)
 end
 
+call{T,N}(::Type{SharedArray{T}}, d::NTuple{N,Int}; kwargs...) =
+    SharedArray(T, d; kwargs...)
+call{T}(::Type{SharedArray{T}}, d::Integer...; kwargs...) =
+    SharedArray(T, d; kwargs...)
+call{T}(::Type{SharedArray{T}}, m::Integer; kwargs...) =
```

Comment:
Just an FYI: this and the next two are just for getting rid of the splatting penalty. For a SharedArray, there's sufficient overhead for creation that this is probably negligible, so these probably aren't really necessary. However, neither do they hurt anything.